README.md

```diff
@@ -43,6 +43,12 @@ Secret goals:
 The backend is accessible at `localhost:3000`, and the frontend at `localhost:5173`.
 Both the backend and the frontend support hot module reloading (HMR).
 
-### Manual
+### Production
 
-- Each part has it's own `README.md` with instructions on how to run it.
+- Install Docker
+- Set the environment variables for the frontend before building (see the [README in ./vinvoor](vinvoor/README.md))
```

Comment:
how does this work when the image is built by github actions?

Reply:
Fixed by [efcf748](https://github.com/ZeusWPI/ZeSS/pull/46/commits/efcf7489a5d27e0307014ff2f5d7f2021c88556a)

Production env variables for the frontend need to be set and kept up to date in production.env
As it is the frontend no confidential information will be stored